notebooks/01_eda/01_inspect_schemas.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think it's worth articulating a) in the `README` and b) at the top of this notebook in a docstring how you envisage the components, datasets and notebooks all fit together

Reply:
Hi, I added the docstring at the top of this notebook, and I noted the task to describe in the `README` how the components fit together in issue #2 .

Generally, I envisage that notebooks are only for exploratory analysis and prototyping. In this case, the notebook was used to check schemas, and to "write down" which columns are of significance (but the notebook in principle is not essential for fetching the dataset).